core/src/test/java/nl/nn/adapterframework/util/XmlUtilsTest.java

```diff
@@ -269,28 +283,77 @@ public void emptyToXhtml() throws Exception {
 
 	@Test
 	public void testConvertEndOfLines() {
-		String input="a\nb\rc\r\nd\r\re\n\nf\r\n\r\ng";
-		String expected="a\nb\nc\nd\n\ne\n\nf\n\ng";
+		String input = "a\nb\rc\r\nd\r\re\n\nf\r\n\r\ng";
+		String expected = "a\nb\nc\nd\n\ne\n\nf\n\ng";
 
 		assertEquals(expected, XmlUtils.convertEndOfLines(input));
 		assertEquals(null, XmlUtils.convertEndOfLines(null));
 	}
 
 	@Test
 	public void testNormalizeWhitespace() {
-		String input="a b  c\td\re\nf\r\n\t\ng";
-		String expected="a b  c d e f    g";
+		String input = "a b  c\td\re\nf\r\n\t\ng";
+		String expected = "a b  c d e f    g";
 
 		assertEquals(expected, XmlUtils.normalizeWhitespace(input));
 		assertEquals(null, XmlUtils.normalizeWhitespace(null));
 	}
 
 	@Test
 	public void testnormalizeAttributeValue() {
-		String input="a b  c\td\re\nf\r\n\t\ng";
-		String expected="a b  c d e f   g";
+		String input = "a b  c\td\re\nf\r\n\t\ng";
+		String expected = "a b  c d e f   g";
 
 		assertEquals(expected, XmlUtils.normalizeAttributeValue(input));
 		assertEquals(null, XmlUtils.normalizeAttributeValue(null));
 	}
+
+	private static Date getCorrectedDate(Date date) {
+		if (CI_TZ.hasSameRules(TEST_TZ)) {
+			return date;
+		} else {
+			Calendar calendar = Calendar.getInstance();
+			calendar.setTime(date);
+			int offset = CI_TZ.getOffset(calendar.getTime().getTime());
+			calendar.add(Calendar.MILLISECOND, -offset);
+			log.info("adjusting date [{}] with offset [{}] to [{}]", () -> date, () -> offset, calendar::getTime);
+			return calendar.getTime();
+		}
+	}
+
+	/**
+	 * Tests have been written in UTC, adjust the TimeZone for CI running with a different default TimeZone
+	 */
+	public static long getCorrectedDate(long l) {
+		Date date = new Date(l);
+		return getCorrectedDate(date).getTime();
+	}
+
+	private static Stream<Arguments> xmlDateTimeData() {
+		return Stream.of(Arguments.of("2013-12-10", 1386633600000L),
```

Comment:
Dit zou je ook in een `@CsvSource` kunnen doen. Dat lijkt op een `@ValueSource` maar dan is iedere waarde in de string array een comma-separated lijst van parameters en JUnit zorgt zelf voor de conversie.